src/core/api/io_services/v1/policy.xml

```diff
@@ -16,29 +16,37 @@
                 <set-backend-service base-url="https://io-p-services-fn-2.azurewebsites.net/api/v1" />
             </when>
             <otherwise>
-                <set-backend-service id="apim-generated-policy" base-url="{{io-fn3-services-url}}/api/v1" />
                 <!-- https://docs.microsoft.com/en-us/dotnet/api/system.random.next?view=net-6.0#system-random-next(system-int32-system-int32) -->
-                <!-- <set-variable name="urlWeight" value="@{
+                <set-variable name="urlWeight" value="@{
```

Comment:
To help reading the policy, can you please add a block comment in plain english to explain the intentions of the following code? Readers must go though implementation details and reverse-engineer the whole algorithm to understand it.